src/pages/Wallet.vue

```diff
@@ -83,44 +83,61 @@ export default defineComponent({
 <template lang="pug">
 q-page
   .row.justify-center
-    q-card
+    q-card.q-mt-xl
       q-card-section
-        .text-h6.text-grey-8
-          | My Balance
+        .text-wallet-heading.text-grey-8
+          | Balance
       q-card-section
-        .text-bold 
-          | {{ balance }} LEGAL
-        .text
-          | {{ gbpValue }} (GBP)
+        .text-wallet-topline.text-bold 
+          | {{ balance }} LCP (LEGALCOIN POUND)
```

Comment:
Should be LEGAL not lcp. Change everywhere.

Reply:
Done